tests/test_swift.py

```diff
@@ -69,44 +69,64 @@ def setUp(self):
         self.channels = set()
         for appInfo in APP_INFOS.values():
             self.channels.update(appInfo.channel)
-        self.channels = sorted(self.channels)
         self.swift = swift.Swift(APP_INFOS)
 
     def test_init(self):
         self.assertEqual(self.swift.appInfos, APP_INFOS)
-        self.assertIsInstance(self.swift.mainWindow, QMainWindow)
-        self.assertIsInstance(self.swift.centralWidget, QLabel)
         for name in APP_INFOS:
             self.assertIn(name, self.swift._dockWidgets)
             self.assertIn(name, self.swift._apps)
         for channel in self.channels:
             self.assertIn(channel, self.swift._subscribers)
 
     def test_app_names(self):
-        appNames = self.swift.appNames()
-        for name in APP_INFOS:
-            self.assertIn(name, appNames)
+        appNamesSet = set(self.swift.appNames())
+        self.assertEqual(appNamesSet, set(APP_INFOS))
+
+    def test_create_app(self):
+        app_ = MagicMock()
+        app_.frames.return_value = (QWidget(),)
+        cls = MagicMock(return_value=app_)
+        setattr(importlib.import_module.return_value, "cls3", cls)
```

Comment:
Can you check if the `import_module()` is called with the argument `"module3"`?

Reply:
Sure! It can be checked just using `import_module.assert_called_with("module3")`.